### PR TITLE
Single pregnancy

### LINKED ISSUE #595: Convert pregnancies to has_one

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

TLDR: Patient-has-many-pregnancies as a way of keeping track of patients who come back is much more of a huge pain than it's worth, so this punts on a whole class of validation and data storage problems in favor of simplicity. We can fix any reporting duplication with group bys, like god intended.

We should take advantage of this in a few other ways, like changing the has to an embed and fixing the routing, in a separate PR. 

BIG s/o to my dog integration tests for confirming that everything still works after this change!

This pull request makes the following changes:
* Converts `Patient has many pregnancies` into a `patient has one pregnancy` 
* Fixes tests and seedfile

It relates to the following issue #s: 
* Fixes #595 
* Bumps #Y

### CODE REVIEW

**Comment by colinxfleming** [comment]
Here goes nothing!
